### ISSUE TITLE
IUIAutomationElement CurrentNativeWindowHandle() method has wrong signature

### ISSUE THREAD

**Issue by AhmadAlHallak** [microsoft/windows-rs#1320]
It seems -if I am not mistaken- that CurrentNativeWindowHandle  (and CachedNativeWindowHandle for the matter) were defined incorrectly to use HWND as the type for retval, where it should be a mut pointer to a HWND.

The current definition renders these methods unusable.

**Comment by riverar** [comment]
@AhmadAlHallak Yep, can reproduce this. Thanks!

(In the future, please provide a minimal reproducible example. This saves us a tremendous amount of time.)

```toml
[package]
name = "app"
version = "0.0.0"
edition = "2018"
publish = false

[dependencies.windows]
version = "0.26.0"
features = [
    "Win32_Foundation",
    "Win32_System_Com",
    "Win32_UI_WindowsAndMessaging",
    "Win32_UI_Accessibility",
]

```

```rust
use windows::runtime::*;
use windows::Win32::System::Com::*;
use windows::Win32::UI::Accessibility::*;
use windows::Win32::UI::WindowsAndMessaging::*;

fn main() -> Result<()> {
    unsafe {
        CoInitializeEx(core::ptr::null_mut(), COINIT_MULTITHREADED)?;
        let uia: IUIAutomation = CoCreateInstance(&CUIAutomation, None, CLSCTX_ALL)?;
        let elem = uia.ElementFromHandle(GetDesktopWindow())?;
        // elem.CurrentNativeWindowHandle(...);
    }

    Ok(())
}
```

**Comment by AhmadAlHallak** [comment]
@riverar Thanks for the quick response.

Will do in the future

**Comment by kennykerr** [comment]
Looks like this was recently fixed in metadata: https://github.com/microsoft/win32metadata/issues/715

Just need them to release another update.

**Comment by AhmadAlHallak** [comment]
> 
> 
> Looks like this was recently fixed in metadata: [microsoft/win32metadata#715](https://github.com/microsoft/win32metadata/issues/715)
> 
> Just need them to release another update.

Ah, this explains why I found the correct definition when searching the metadata repo

Should this issue be closed since it's not an issue with the windows crate itself?

**Comment by riverar** [comment]
Yep, sounds good. Thanks again!
